recipe/conda_build_config.yaml

```diff
@@ -343,6 +345,8 @@ jasper:
   - 1.900.1
 jpeg:
   - 9
+libjpeg-turbo
```

Comment:
Think this needs to be an underscore like `json_c` below.

Reply:
Why it is not the package name? If that is a yaml thing we should fix that and not workaround it by creating confusing `-` and `_` changes instead.

Reply:
It's a conda-build thing. these variables are used as jinja variables and env variables and therefore `-` is not allowed.

Reply:
:-(

Thanks for the info @isuruf!